templates/partials/committee-charts.html

```diff
@@ -1,3 +1,6 @@
+{% with %}
```

Comment:
Ah, you're right. Part of the problem is that we're using different data structures for the candidate and committee data--there's an extra `financials` key in the committee data when it's fetched as part of the candidate endpoint
* Fetch a committee directly: `committee.reports`
* Fetch a committee nested within a candidate: `committee.financials.reports`

I think if we use consistent data structures for nesting financial statements within committees, and reintroduce the `{% with committee = context() %} in `committees-single.html` above, this template would work in both uses.